gems/aws-sdk-s3/spec/object/upload_file_spec.rb

```diff
@@ -44,9 +44,9 @@ module S3
           end
         end
 
-        let(:seventeen_meg_file) do
+        let(:one_hundred_seventeen_meg_file) do
           Tempfile.new('one-meg-file').tap do |f|
```

Comment:
Tempfile name should probably be changed.

Reply:
Noticed the 10 MB file also had a stale identifier, so I've changed that as well.